pkg/cloud/scope/machine.go

```diff
@@ -260,7 +260,10 @@ func (m *MachineScope) GetRawBootstrapData() ([]byte, error) {
 
 // PatchObject persists the machine spec and status.
 func (m *MachineScope) PatchObject() error {
-	return m.patchHelper.Patch(context.TODO(), m.AWSMachine)
+	return m.patchHelper.Patch(
+		context.TODO(),
+		m.AWSMachine,
+		patch.WithOwnedConditions{Conditions: []clusterv1.ConditionType{infrav1.InstanceReadyCondition, infrav1.SecurityGroupsReadyCondition, infrav1.ELBAttachedCondition}})
```

Comment:
nit (not blocking)
split this line into multiple lines for better visibility

Reply:
Thanks. Updated